pkg/printers/internalversion/printers_test.go

```diff
@@ -886,39 +886,171 @@ func TestPrintNodeInternalIP(t *testing.T) {
 }
 
 func TestPrintIngress(t *testing.T) {
```

Comment:
FYI, this diff looks pretty bad, but what I did here was turn a single test into a test with multiple test cases.